editor/import/resource_importer_obj.cpp

```diff
@@ -378,12 +380,13 @@ static Error _parse_obj(const String &p_path, List<Ref<Mesh>> &r_meshes, bool p_
 					surf_tool->generate_normals();
 				}
 
+				//force triangle pairs to share tangents
```

Comment:
```suggestion
				// Force triangle pairs to share tangents.
```
Here also, sorry missed this one

Reply:
I added periods to the other comments, but made this one consistent with the style of the surrounding comments ¯\\\_(ツ)\_/¯

What should be preferred in this scenario?

Reply:
New comments or comments edited should be made to follow the code style, other nearby comments shouldn't matter, though we don't edit them to avoid touching code that's out of scope